tests/unit/memefficiency.tcl

```diff
@@ -37,9 +37,10 @@ start_server {tags {"memefficiency external:skip"}} {
 }
 
 run_solo {defrag} {
-start_server {tags {"defrag external:skip"} overrides {appendonly yes auto-aof-rewrite-percentage 0 save ""}} {
+    proc test_active_defrag {type} {
```

Comment:
so now all the defrag tests are running twice?
isn't that a little excessive?
if we keep it, or find a better one, do we want to reflect that change in unstable?

Reply:
but these two are not the same, one is for cluster, and another is for standalone.

Reply:
i know.
i'm saying that in the past we had a bunch of tests: [plain, aof, large keys, edge case, eval]
and now we run nearly all of them twice.
i think it's excessive, it's probably enough to just run one or two of them twice, or just add a dedicated test for cluster defrag (which i think we already have in unstable)

Reply:
> or just add a dedicated test for cluster defrag (which i think we already have in unstable)

i'll do it.

Reply:
@sundb so to be sure: with this PR getting merged, the tests in both 7.2 and 7.4 are now able to detect a bug similar to the one being fixed here (which doesn't exist in 7.4)?

Reply:
ohh, now i actually looked at the test code:
```
                # It repeatedly enables and disables active defragmentation,
                # and checks if it crashes, see issue #13307.
```

well, i suppose it's pointless to test the `expires_cursor` reset thing, since that code is now simplified.

and the `flushdb async` bug isn't really related to the defrag, so if we wanted, we could have added another simpler explicit check for it.
truth be told, that emptyDbAsync does have some duplicated logic in it that can someday get out of sync.
but i suppose that's completely unrelated to this PR.